Enable microphone on Mac

mself opened the pull request:
- Create a custom Info.plist with the NSMicrophoneUsageDescription key
- Update pitch.pro to copy the Info.plist into the app budle
- Also set the app icon to be the original pitch.icns file

quicky2000 commented:
Hi,

Coudl it be possible to add Info.plist to DISTFILES only for Mac OS configuration ?
What about CMake build ?

mself commented:
I removed `Info.plist` from `DISTFILES` and also moved the Mac-specific files into a new `mac` folder.

It verifed that it builds correctly for:
- QtCreator (IDE)
- Xcode (IDE)
- qmake + make (command line)

I don't use CMake, so I don't plan to make any changes for that (if any are needed).  There are already three different ways to build the project, so why would you use CMake?

quicky2000 commented:
> I removed `Info.plist` from `DISTFILES` and also moved the Mac-specific files into a new `mac` folder.
> 
> It verifed that it builds correctly for:
> 
>     * QtCreator (IDE)
> 
>     * Xcode (IDE)
> 
>     * qmake + make (command line)
> 
> 
> I don't use CMake, so I don't plan to make any changes for that (if any are needed). There are already three different ways to build the project, so why would you use CMake?

You perhaps didn't remark it but on Travis-Ci perform is build with both QMake and CMake on both Linux and MacOs.
I use CMake because this is mandatory for my IDE (CLion), mutli-platform, widely supported in software industry. Anyway I can search to reflect what you did in Qmake on CMake
